sites.json

```diff
@@ -1857,6 +1857,16 @@
     },
 
     {
+        "name": "ESPN",
+        "url": "http://www.espn.com",
```

Comment:
Can this become the link to the account settings page?

Reply:
Site uses some sort of Javascript popup window and the base URL remains the same at www.espn.com

Reply:
Yuck, ok then. Nothing we can do there, I guess.